lib/scrabble.rb

```diff
@@ -1,9 +1,37 @@
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
```

Comment:
Now that I think of it, I see that 'word' isn't necessarily an attribute/state of Scrabble. I think initially I thought it was an element of Scrabble so I thought it should be an instance variable.